docs/changelog.rst

```diff
@@ -7,6 +7,11 @@ Changes in nbformat
 In Development
 ==============
 
+5.1.2
```

Comment:
5.1.2 is already released. The next one would be 5.1.3.

Reply:
Good catch. I'll fix that later this afternoon.